src/turndown.js

```diff
@@ -352,6 +352,23 @@ turndownService.addRule("edu_breakdown", {
   }
 })
 
+/**
+ * Remove Classroom section
+ **/
+turndownService.addRule("remove_the_classroom", {
+  filter: (node, options) => {
+    if (node.nodeName !== "DIV") return false
+    const children = Array.from(node.childNodes)
+    return children.some(node => {
+      if (node.nodeName !== "A") return false
+      return node.getAttribute("name") === "classroom"
```

Comment:
could this be `node.nodeName === "A" && node.getAttribute("name") === "classroom"` ? that sort of phrases the condition positively, if that makes sense